oauth2client/contrib/django_util/__init__.py

```diff
@@ -160,19 +174,59 @@ def drive_required(request):
    from oauth2client.contrib.django_util.signals import oauth2_authorized
 
    def test_callback(sender, request, credentials, **kwargs):
-       print "Authorization Signal Received %s" % credentials.id_token['email']
+       print("Authorization Signal Received %s"
+           % credentials.id_token['email'])
 
    oauth2_authorized.connect(test_callback)
 
+Adding Credentials To An Existing Django User System
+=====================================================
+
+As an alternative to storing the credentials in the session, the helper
+can be configured to store the fields on a Django model. This might be useful
+if you need to use the credentials outside the context of a user request, and
+prevents the need for a logged in user to repeat the OAuth flow when starting a
+new session.
+
+To use, change `settings.py`
+
+.. code-block:: python
+   :caption:  settings.py
+   :name: secrets_file
+
+   GOOGLE_OAUTH2_STORAGE_MODEL = {
+    'model': 'path.to.model.MyModel',
+    'user_property': 'user_id',
+    'credentials_property': 'credential'
+    }
+
+Where path.to.model class is the fully qualified name of a django.db.model
+class containing a `django.contrib.auth.models.User` field with the
+ name specified by `user_property` and a
+`oauth2client.contrib.django_util.models.CredentialsField` with the name
+specified by `credentials_property`. For the sample configuration given,
+our model would look like
```

Comment:
Use `::` and indent the sample below two blocks to make it format it as a code block.